Gopkg.lock

```diff
@@ -4,18 +4,18 @@
 [[projects]]
   branch = "master"
   name = "github.com/WICG/webpackage"
-  packages = [
-    "go/signedexchange/cbor",
-    "go/signedexchange/certurl",
-    "go/signedexchange/mice"
-  ]
+  packages = ["go/signedexchange/certurl"]
```

Comment:
You might want to have this reference github.com/nyaxt/webpackage too in the near future. The WICG repo will be updated to use the new certURL format.

Reply:
Oops, yes, done.